src/components/search.js

```diff
@@ -59,7 +59,7 @@ const RepoSearch = () => {
             return <div className="repocard" key={repo.id}>{repo.name}</div>
           }) :
           filterList.map(repo => {
-            return <div key={repo.id}>{repo.name}</div>
+            return <div className="repocard" key={repo.id}>{repo.name}</div>
```

Comment:
This change isn't needed because I modified the structure of this .map, it isn't up-to-date with current master. This className was used in another component on a different file (RepoListItem.js).

Reply:
Merge conflict resolved. Threw out old code from Search.js. Thanks.